dap_job_quality/pipeline/prodigy/sentence_classifier_recipe.py

```diff
@@ -0,0 +1,157 @@
+"""
+Usage:
+
+To start the app:
+```
+prodigy job_ad_sent_cat job_sentences_sample ../../../inputs/labelling/job_sentences.jsonl -F sentence_classifier_recipe.py
+```
+
+To export the labelled data locally:
```

Comment:
If we continue to work locally, I had to create the directory first
```suggestion
To export the labelled data locally:
mkdir -p ../../../inputs/labelled
```